src/components/tablePropertyButtonWithPopover/SortAndFilterBtnsContent.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-debugger */
+import React from "react";
+import PropTypes from "prop-types";
+import NorthIcon from "@mui/icons-material/North";
+import SouthIcon from "@mui/icons-material/South";
+import FilterListIcon from "@mui/icons-material/FilterList";
+import { useDispatch } from "react-redux";
+import style from "./popoverOfButton.module.scss";
+import { sortDataToAscendingOrDescending } from "../../features/tableDataInfo/tableDataInfoSlice";
+import sortedPositions from "./sortAndFilterFeatures";
+
+function SortAndFilterBtnsContent({ item, setAnchorElement }) {
+  const dispatch = useDispatch();
+
+  const handleSortToAscending = () => {
+    dispatch(
+      sortDataToAscendingOrDescending({
+        positions: sortedPositions(item, "ascending"),
+      })
+    );
+    setAnchorElement(false);
+  };
+
+  const handleSortToDescending = () => {
+    dispatch(
+      sortDataToAscendingOrDescending({
+        positions: sortedPositions(item, "descending"),
+      })
+    );
+    setAnchorElement(false);
+  };
```

Comment:
these 2 functions have only one small difference, try to merge them

Reply:
+